examples/ex5p.cpp

```diff
@@ -326,7 +326,16 @@ int main(int argc, char *argv[])
                       DataCollection::PARALLEL_FORMAT);
    visit_dc.Save();
 
-   // 16. Send the solution by socket to a GLVis server.
+   // 16. Save data in the Paraview format
+   ParaviewDataCollection paraview_dc("PWExample5", pmesh);
```

Comment:
Update the "Description" section in the beginning to mention also Paraview?

Reply:
I will check example 9  - I will update the comments also.